storage/database/db_manager.go

```diff
@@ -1057,30 +1162,100 @@ func (dbm *databaseManager) WriteMerkleProof(key, value []byte) {
 
 // Cached Trie Node operation.
 func (dbm *databaseManager) ReadCachedTrieNode(hash common.Hash) ([]byte, error) {
+	if dbm.inMigration {
+		if val, err := dbm.newStateTrieDB.Get(hash[:]); err == nil {
```

Comment:
What happened if `err != nil`?

Reply:
@jeonghonah Then it will go to line 1170 and return the result from `ReadCachedTrieNodeFromOld`.

Reply:
This is because when there is no value found, LevelDB returns nil value and `NotFound` error.